client.go

```diff
@@ -221,35 +222,28 @@ func (r *marathonClient) apiDelete(uri string, post, result interface{}) error {
 }
 
 func (r *marathonClient) apiCall(method, uri string, body, result interface{}) error {
+	var jsonBody []byte
```

Comment:
FWIW, they say you should declare variables as close to their first usage as possible. :-)